crew

```diff
@@ -355,7 +355,7 @@ def const (var)
       'CREW_LIB_PREFIX',
       'CREW_LIB_SUFFIX',
       'CREW_MAN_PREFIX',
-      'CREW_MESON_LTO_OPTIONS',
```

Comment:
Why was this removed?

Reply:
No longer should be removed. https://github.com/skycocker/chromebrew/pull/5593#issuecomment-817401138

Reply:
You mean no longer should be used?

Reply:
Fixed.